lib/src/rules/prefer_initializing_formals.dart

```diff
@@ -88,14 +88,14 @@ Iterable<ConstructorFieldInitializer>
             ConstructorDeclaration node) =>
         node.initializers.whereType<ConstructorFieldInitializer>();
 
-Element _getLeftElement(AssignmentExpression assignment) =>
+FieldElement _getLeftElement(AssignmentExpression assignment) =>
```

Comment:
I don't think this is a safe change to make. This method can be called with any assignment in a constructor and the left hand side won't necessarily resolve to a field. For example, it might be an assignment to a local variable.

Reply:
Indeed in fact I did not needed to change this one!